admin/resources/js/components/UpdateUser.tsx

```diff
@@ -91,22 +93,38 @@ export const UpdateUserForm: React.FunctionComponent<UpdateUserFormProps> = ({
   );
 };
 
-export const UpdateUser: React.FunctionComponent<{ initialUser: User }> = ({
-  initialUser,
+export const UpdateUser: React.FunctionComponent<{ userId: string }> = ({
+  userId,
 }) => {
+  const [{ data: userData, fetching, error }, _reexecuteUser] = useUserQuery({
+    variables: { id: userId },
+  });
+
   const [_result, executeMutation] = useUpdateUserMutation();
```

Comment:
'_result' is assigned a value but never used.

Reply:
I've seen people use `_` or names starting with `_` to for unused variables in destructuring assignments, but it turns out you can just leave the unused variable out entirely: `const [, executeMutation] = useUpdateUserMutation();`